tutorials/parachain-to-polkadot-vault/1-step1.mdx

```diff
@@ -0,0 +1,35 @@
+# Introduction
+
+## Polkadot Vault
+Polkadot Vault is a mobile application that allows any smartphone to act as an air-gapped crypto wallet.
+This is also known as "cold storage".
+
+You can create accounts in Substrate-based networks, sign messages/transactions,
+and transfer funds to and from these accounts without any sort of connectivity enabled on the device.
+
+Any data transfer from or to the app happens using QR code.
+By doing so, the most sensitive piece of information, the private keys, will never leave the phone.
+
+**Note:** This is not a complete crypto wallet in itself. The Vault does not sync with blockchain,
+so it does not know your account balance, whether transactions were successful or even if the account exists!
+This is a cold wallet app that only stores keys, reads and signs messages. It should always be used with hot wallet like [polkadot.js](https://polkadot.js.org/extension/).
+
+GitHub repo: https://github.com/paritytech/parity-signer
+
+## Metadata Portal
```

Comment:
In this section it would be nice to have some context. Why do we need the metadata portal? (to give descriptions to users before signing a tx?)